### PR TITLE
Exit proxy gracefully with SIGTERM

### PR DESCRIPTION
**Pull request by rcasta74**
Fix jasonacox/Powerwall-Dashboard#353

### CODE REVIEW

File: proxy/server.py [hunk 1]
```diff
@@ -344,6 +346,11 @@ def do_GET(self):
         except:
             log.error("Socket broken sending response [doGET]")
 
+def sigTermHandle(signum, frame):
+    sys.exit()
```
Comment (mcbirse) [inline proxy/server.py:350]: My recommendation/preference would be to use `raise SystemExit` per explanation [here](https://github.com/jasonacox/Powerwall-Dashboard/issues/353?notification_referrer_id=NT_kwDOBnsFf7Q3NzcxNzQzMTA2OjEwODcyNTYzMQ#issuecomment-1728709935)

**Review by mcbirse — CHANGES_REQUESTED** [review]

**Comment by rcasta74** [comment]
I will not have access to my pc for few days, feel free to update the commit by yourself.

On Saturday, September 23, 2023, Michael Birse wrote:
> @mcbirse requested changes on this pull request.
> 
> 
> 
> > @@ -344,6 +346,11 @@ def do_GET(self):
>          except:
>              log.error("Socket broken sending response [doGET]")
>  
> +def sigTermHandle(signum, frame):
> +    sys.exit()
> 
> My recommendation/preference would be to use `raise SystemExit` per explanation [here](https://github.com/jasonacox/Powerwall-Dashboard/issues/353?notification_referrer_id=NT_kwDOBnsFf7Q3NzcxNzQzMTA2OjEwODcyNTYzMQ#issuecomment-1728709935)
> 
> -- 
> Reply to this email directly or view it on GitHub:
> https://github.com/jasonacox/pypowerwall/pull/49#pullrequestreview-1640788088
> You are receiving this because you authored the thread.
> 
> Message ID: ***@***.***

-- 
Sent from my Sailfish device

File: proxy/server.py [hunk 2]
```diff
@@ -24,6 +24,8 @@
 import logging
 import resource
 import datetime
+import signal
+import sys
```
Comment (mcbirse) [inline proxy/server.py:28]: `import sys` not required if using `raise SystemExit` in SIGTERM handler

**Comment by jasonacox** [comment]
Thanks!! 🙏

**Pull request merged**